packages/interactive_media_ads/lib/src/ads_loader.dart

```diff
@@ -92,6 +93,10 @@ class AdsLoader {
   }
 
   /// Requests ads from a server.
+  ///
+  /// Note that ads cannot be request until the `AdDisplayContainer` has been
```

Comment:
Just "Ads cannot be [...]", per Flutter style guide, per https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-empty-prose